pack/valk_encounter.json

```
@@ -34,13 +37,15 @@
 		"boost": 2,
 		"code": "25030",
 		"faction_code": "encounter",
+		"flavor": "Behind Amora's charm lies a cruel ambition and the power to subdue those in her way.",
 		"name": "Powerful Enchantments",
 		"octgn_id": "c5cf7bc4-2886-4ed5-a883-437a3d2b8356",
 		"pack_code": "valk",
 		"position": 30,
 		"quantity": 1,
```

I believe this side scheme is missing the following data to about the acceleration:

```suggestion
		"quantity": 1,
		"scheme_acceleration": 1,
```